### DISCUSSION TITLE
Do RAM Memory frequency matter?

### DISCUSSION THREAD

**Discussion by rHilkner** [madMAx43v3r/chia-plotter#175] [Q&A]
I have a computer with 5950x (16 cores, 32 threads), 64gb RAM 3000mhz and 6tb trashy nvme drives (980 [not pro] and crucial p2). I was about to make an upgrade and buy 4-6x 970 pro, but after madmax plotter I'm thinking about going to 128gb RAM to make a 110gb ramdisk instead (or would 256gb be better?). So I'm wondering how much does RAM frequency impacts the performance of the plotting? I want to make a real high-end plotter here, just wondering what would be the best configs for my system. I may also save some money by buying 128gb RAM instead of 6x 970 pro.

**Comment by madMAx43v3r** [comment]
stable RAM is more important, get some cheap RAM and don't overclock it

**Reply by rHilkner** [reply]
So 2400mhz vs 3600mhz shouldn't make much difference? and 128gb ram is enough right?

**Reply by madMAx43v3r** [reply]
yeah 2400 is fine, capacity is more important, 128GB is the minimum, it'll fit 16 threads besides the ramdisk, just enough

**Reply by madMAx43v3r** [reply]
well actually since you already got 64GB@3000mhz, i would match that with another 64GB@3000

**Reply by adamsky-01** [reply]
> yeah 2400 is fine, capacity is more important, 128GB is the minimum, it'll fit 16 threads besides the ramdisk, just enough

Will 128GiB ram handle the 104 GiB ram disk and 16 threads, not max. 12 threads? 
https://github.com/madMAx43v3r/chia-plotter/discussions/150#discussioncomment-854256

**Reply by Kiggles** [reply]
Yes, with fewer threads, less ram is required. 12 threads, opposed to 16, will fit in 128GB total even easier.

**Reply by rHilkner** [reply]
How much RAM is needed per thread?

**Comment by hajes** [comment]
frequency doesn't matter as much as timing. Speed is only part of the story. Tight timing "slow" RAM will be faster than high speed loose timing. RAM timing can be optimized if you have about 1 month, and legendary tuning/tweaking skills/experience...you gain up to 10% faster RAM

fastest RAM sets you back about 1000CHF...in my world, not worthy to waste 50% of resources for 5% gain...what might be worthy is slow 256RAM, and plot two plots parallel...with 16 threads split...you lose time per plot, and perhaps you gain overall 

My workstation (Ryzen 3950X, 128GB cheap G.Skill crap, Corsair MP600 2TB, ASUS ArtPro B550-Creator) does 1 plot/~23 min/~70W @ 0.05CHF/plot; over 60 plots per day.

EFFICIENCY and resale is the key...super workstation for 10k is unsellable afterwards...simple tourist 2k PC, will be always sold for at least 50% of original price...I even bought used, resold later for more ;-)
